### ISSUE TITLE
Update `no invalid role` rule to check against valid role values

### ISSUE THREAD

**Issue by MelSumner** [ember-template-lint/ember-template-lint#1311]
Update the `no-invalid-role` rule to check against the permitted role values listed at https://www.w3.org/WAI/PF/aria/roles and ensure that the rule catches `role` attribute values that are not valid. 

Disallowed:
```html
<div role="accordion"></div>
```
```html
<div role="command interface"></div>
```

Allowed: 
```html
<footer role="contentinfo"></footer>
```
```html
<div role="log"></div>
```

**Comment by rajasegar** [comment]
I am picking this one, @MelSumner Can you please assign it to me?
